src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -22,7 +22,7 @@
             <label id="company-label">Name</label>
           </v-col>
           <v-col cols="12" sm="9" class="pt-4 pt-sm-0">
-            <div id="company-name">{{ getNameRequestApprovedName || '[Incorporation Number] B.C. Ltd.' }}</div>
+            <div id="company-name">{{ getNameRequestApprovedName || '[Incorporation Number] B.C. LTD.' }}</div>
```

Comment:
Everywhere else it's "LTD" so this is a fix. (It's also now consistent with the name components I'm porting over from Edit UI.)